nibabel/pydicom_compat.py

```diff
@@ -40,6 +40,7 @@
     read_file = pydicom.read_file
 
 if have_dicom:
+    from pydicom.sequence import Sequence
```

Comment:
Does this work with `pydicom < 1.0`? Importing doesn't respect renamed modules, so I suspect this is going to need to go up in the `else` blocks (L35-L40).